beetmoverscript/test/test_utils.py

```diff
@@ -218,6 +487,32 @@ def test_beetmover_template_args_generation_release(context):
     assert template_args == expected_template_args
 
 
+@pytest.mark.parametrize('branch, version, artifact_id, build_id, expected_version', ((
+    'mozilla-central', '63.0a1', 'geckoview-nightly-x86', '20181231120000', '63.0.20181231120000',
+), (
+    'mozilla-beta', '63.0b2', 'geckoview-beta-armeabi-v7a', '20181231120000', '63.0b2',
+), (
+    'mozilla-release', '63.0', 'geckoview-arm64-v8a', '20181231120000', '63.0',
+)))
```

Comment:
nit: might be worth adding something for release/esr as well here, just for the sake of consistency?

Reply:
Right. For the record, ESR is not supported as geckoview is an android build and Android deactivated on ESR..